python/sglang/srt/multimodal/processors/base_processor.py

```diff
@@ -324,6 +333,8 @@ def process_mm_data(
             }:
                 # Note: for qwen-vl, processor has some reshape issue because of dims restriction on Ascend.
                 kwargs["device"] = "npu"
+            if _is_cpu:
+                kwargs["device"] = "cpu"
```

Comment:
move it before line 328, like this:
https://github.com/sgl-project/sglang/pull/12662/files#diff-79b2e61a4268a86febebb182d41769931254d2356eb9ca31ebf652bcb893fe8eR259-R261